Cargo.toml

```diff
@@ -15,7 +15,9 @@ members = [
     "spartan",
     "core",
     "jolt",
-    "sdk",
+    "sdk", 
+    "macro",
```

Comment:
Why isn't this part of the runtime (`nexus_rt_macros` exists)?

Reply:
This macro is supposed to be used in host, focus on the development/optimization process. 
So I think it's better to have its own crate separate from the nexus_runtime_macros. 
I don't think of a good name for this, I can call it "nexus-host-macros" to separate from "nexus-rt-macros", eventually I put the name is `nexus-macro`.

Reply:
This macro is supposed to be used in host, focus on the development/optimization process. 
So I think it's better to have its own crate separate from the nexus_runtime_macros. 
I don't think of a good name for this, I can call it "nexus-host-macros" to separate from "nexus-rt-macros", eventually I put the name is `nexus-macro`.

Reply:
This macro can also be used outside of the SDK (since SDK exposed a limited API), I included this macro as `dev-dependencies` for the SDK, I think it's better to live outside of the SDK. What do you think?

Reply:
That's a fair point, you can leave as is then.